internal/helper/helper.go

```diff
@@ -99,6 +99,12 @@ func GetCopyright(content string) string {
 		if strings.Contains(strings.ToLower(tokens[0]), "copyright") {
 			return line
 		}
+		for  _,l :=  range lines {
```

Comment:
Is it needed? if yes, then can we please combine. The func is doing the redundant  checks